src/main/java/io/codelex/arrays/practice/TicTacToe.java

```diff
@@ -1,17 +1,80 @@
 package io.codelex.arrays.practice;
-
 import java.util.Scanner;
 
 public class TicTacToe {
 
-    private static char[][] board = new char[3][3];
+    static char[][] board = new char[3][3];
```

Comment:
šis jau varēja palikt private static final :thinking:

Reply:
Paldies, izlaboju.